modules/custom/activity_send/modules/activity_send_email/js/collapse.js

```diff
@@ -0,0 +1,24 @@
+(function ($, Drupal) {
+
+  Drupal.behaviors.socialActivitySendEmail = {
+    attach: function attach(context) {
+      var $context = $(context);
+
+      $context.find('.form-email-notification .card__title > a').each(function () {
+        $(this).on('click', function () {
+          var $icon = $(this).find(' svg');
+
+          if ($icon.hasClass('icon-expand_more')) {
+            $icon.html('<use xlink:href="#icon-expand-less" />');
```

Comment:
I am not sure but maybe will be better to change just the **xlink:href** attribute value.